css/style.css

```diff
@@ -0,0 +1,438 @@
+body {
+  font-size: 0.8rem;
+  font-family: "Roboto", Arial, sans-serif;
+}
+
+hr {
+  border: 1px solid #e3e3e3;
+}
+
+a {
+  text-decoration: none;
+}
+
+/* flex box */
+.flex{
+    display: flex;
+}
+.flex-row{
+  flex-direction: row;
+}
+
+.flex-column{
+  flex-direction: column;
+}
+
+.nav-bar {
+  justify-content: space-between;
+  height: 36px;
+  margin: 20px;
+}
+
+.col1 {
+  justify-content: flex-start;
+  margin-right: 15%;
+}
+
+.col1 > .logo {
+  width: 80px;
+  height: 24px;
+  margin-top: 10px;
+}
+
+.col2 {
+  flex: 0.7;
+  margin-top: 10px;
+  justify-content: center;
+}
+
+.col2 > form {
+  flex: 1;
+  height: 32px;
+}
+
+.col2 .searchbox {
+  margin-right: 0;
+  padding-right: 0;
+  flex-grow: 0.7;
+  border-style: solid;
+  border-radius: 0;
+  border-right: 0;
+  box-shadow: none !important;
+}
+
+.col2 .searchBtn {
+  margin-left: 0 !important;
+  border-style: solid;
+  border-radius: 0;
+  cursor: pointer;
+}
+
+.comments-icon > svg {
+  width: 24px;
+  height: 24px;
+  fill: #5c5a5a;
+}
+
+.save-icon > a > i > svg {
+  width: 20px;
+  height: 20px;
+  fill: #666464;
+  vertical-align: middle;
+  display: inline-block;
+}
+
+.col3 .icon-wrapper svg {
+  fill: #606060;
+}
+
+.col2 .searchBtn svg {
+  width: 25px;
+  height: 25px;
+  fill: rgb(165, 161, 161);
+}
+
+.col2 .searchBtn svg:hover {
+  fill: rgb(122, 116, 116);
+}
+
+.col3 {
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.col3 .icon-wrapper .avatar {
+  border-radius: 60px;
+}
+
+.icon-wrapper {
+  width: 24px;
+  height: 24px;
+  padding: 0 15px;
+}
+
+.page-wrapper {
+  flex-direction: row;
+  background: #f8f8fa;
+  padding: 20px;
+}
+
+.main-content {
+  width: 70%;
+  margin: 20px 20px;
+}
+
+/* video stats */
+.video-stats {
+  justify-content: space-between;
+}
+
+.views {
+  margin: 10px;
+}
+
+.likes-dislikes > div > button,
+.likes-dislikes > div > a,
+.likes-dislikes > .likes {
+  margin: 5px;
+  background: transparent;
+  border: none;
+}
+
+.likes-dislikes > .save-icon > button {
+  margin: 5px;
+  background: transparent;
```

Comment:
You guys already have a class `.bg-transparent` that deals with making the background transparent, remove it from your code and use that class in your HTML file whenever necessary